diff --git a/src/context.rs b/src/context.rs
index 9469ea17f..96329133a 100644
--- a/src/context.rs
+++ b/src/context.rs
@@ -125,9 +125,16 @@ impl Parse for ContextItems {
     Ok(Self {
       config_file: config_file
         .unwrap_or_else(|| {
-          std::env::var("CARGO_MANIFEST_DIR")
-            .map(|m| PathBuf::from(m).join("tauri.conf.json"))
+          // Returns the path from 'TAURI_CONF_JSON' if available, otherwise
+          // creates a path like 'CARGO_MANIFEST_DIR/tauri.conf.json'
+          std::env::var("TAURI_CONF_JSON")
+            .map(|m| PathBuf::from(m))
             .map_err(|e| e.to_string())
+            .or(
+              std::env::var("CARGO_MANIFEST_DIR")
+                .map(|m| PathBuf::from(m).join("tauri.conf.json"))
+                .map_err(|e| e.to_string()),
+            )
         })
         .map_err(|e| input.error(e))?,
       root: root.unwrap_or_else(|| {
