diff --git a/toolchain/cc_toolchain_config.bzl b/toolchain/cc_toolchain_config.bzl
index 434345f..6a6e68a 100644
--- a/toolchain/cc_toolchain_config.bzl
+++ b/toolchain/cc_toolchain_config.bzl
@@ -151,7 +151,11 @@ def cc_toolchain_config(
         use_lld = False
         link_flags.extend([
             "-headerpad_max_install_names",
-            "-fobjc-link-runtime",
+            # This will issue a warning on macOS ventura; see:
+            #  https://github.com/python/cpython/issues/97524
+            #  https://developer.apple.com/forums/thread/719961
+            "-undefined",
+            "dynamic_lookup",
         ])

         # Use the bundled libtool (llvm-libtool-darwin).
